components/Room-Components/UsersList.js

```diff
@@ -1,36 +1,19 @@
 import React, { Component } from 'react';
 import { Text, View, ScrollView, TouchableOpacity, Alert } from 'react-native';
+import * as GLOBAL from '../../state/global';
 
 import { styles } from '../../styles/styles';
 
 class UsersList extends Component {
-    state = {
-        isHost: true,
-        userList: [
-            "user1",
-            "user2",
-            "user3",
-            "user4",
-            "user5",
-            "user6",
-            "user7",
-            "user8",
-            "user9",
-            "user10",
-            "user11",
-            "user12",
-            "user13",
-        ]
-    };
 
-    kickUser = (User) => {
-        console.log("Kick" + User);
+    kickUser = (user) => {
+        GLOBAL.roomServiceSocket.emit('kick_user', { roomID: GLOBAL.roomName, user, authToken: GLOBAL.authToken });
     }
 
     kickButtonPressed = (User) => {
         Alert.alert(
             " ",
-            "Kick " + User + "?",
+            "Kick " + User.nickname + "?",
```

Comment:
Is `User` a variable or a class? Seems like a variable but named like a class.
This might be a class component thing I'm not aware of since I only use function components atm.

Reply:
Nah, I just left it like that since it was there originally. I'll rename it to be more like a variable.